src/gui/sharelinkwidget.cpp

```diff
@@ -79,11 +79,10 @@ ShareLinkWidget::ShareLinkWidget(AccountPtr account,
     connect(_ui->confirmPassword, &QAbstractButton::clicked, this, &ShareLinkWidget::slotCreatePassword);
     connect(_ui->confirmNote, &QAbstractButton::clicked, this, &ShareLinkWidget::slotCreateNote);
     connect(_ui->confirmExpirationDate, &QAbstractButton::clicked, this, &ShareLinkWidget::slotSetExpireDate);
-    connect(_ui->calendar, &QDateTimeEdit::dateChanged, this, &ShareLinkWidget::slotSetExpireDate);
 
     _ui->errorLabel->hide();
 
-    bool sharingPossible = true;
+    auto sharingPossible = true;
```

Comment:
I was never able to find benefits of replacing POD type names with auto :)

Reply:
Don' repeat yourself :)

Reply:
@FlexW This does not go that far to trigger the DRY ;)